test/unit/gateways/aafes_test.rb

```diff
@@ -0,0 +1,186 @@
+      -> "HTTP/1.1 200 OK\r\n"
+      -> "Connection: close\r\n"
+      -> "X-Powered-By: Undertow/1\r\n"
+      -> "Server: WildFly/10\r\n"
+      -> "Content-Type: application/xml;charset=UTF-8\r\n"
+      -> "Content-Length: 925\r\n"
+      -> "Date: Thu, 11 Jun 2020 12:33:42 GMT\r\n"
+      -> "\r\n"
+      reading 925 bytes...
+      -> "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<Message TypeCode=\"Response\" MajorVersion=\"3\" MinorVersion=\"4\" FixVersion=\"0\" xmlns=\"http://www.aafes.com/credit\">\n    <Header>\n        <IdentityUUID>9765830b-38ec-4154-b349-15ef4a302489</IdentityUUID>\n        <LocalDateTime>2019-09-04T22:26:02</LocalDateTime>\n        <SettleIndicator>false</SettleIndicator>\n        <OrderNumber>ONP3951033</OrderNumber>\n        <transactionId>6750</transactionId>\n        <termId>20</termId>\n        <Comment>Test</Comment>\n        <CustomerID>45017632990</CustomerID>\n    </Header>\n    <Response RRN=\"RRNPGtwi5361\">\n        <Media>Milstar</Media>\n        <ResponseType>Approved</ResponseType>\n        <AuthNumber>020503</AuthNumber>\n        <ReasonCode>000</ReasonCode>\n        <PlanNumber>10001</PlanNumber>\n        <DescriptionField>APPROVED  </DescriptionField>\n        <origReqType>Sale</origReqType>\n    </Response>\n</Message>\n"
+      read 925 bytes
+      Conn close
+    TRANSCRIPT
+  end
+
+  # def post_scrubbed
+  # end
+
+  def successful_authorize_response
+    <<-XML
+    <Message TypeCode="Response" MajorVersion="3" MinorVersion="4" FixVersion="0" xmlns="http://www.aafes.com/credit">
+      <Header>
+          <IdentityUUID>9765830b-38ec-4154-b349-15ef4a302489</IdentityUUID>
+          <LocalDateTime>2020-06-11T19:20:10Z</LocalDateTime>
+          <SettleIndicator>false</SettleIndicator>
+          <OrderNumber>ONP3951033</OrderNumber>
+          <transactionId>6750</transactionId>
+          <termId>20</termId>
+          <Comment>Test</Comment>
+          <CustomerID>45017632990</CustomerID>
+      </Header>
+      <Response RRN="RRNP45805353">
+          <Media>Milstar</Media>
+          <ResponseType>Approved</ResponseType>
+          <AuthNumber>020588</AuthNumber>
+          <ReasonCode>000</ReasonCode>
+          <PlanNumber>10001</PlanNumber>
+          <DescriptionField>APPROVED  </DescriptionField>
+          <origReqType>Sale</origReqType>
+      </Response>
+    </Message>
+    XML
+  end
+
+  def failed_authorize_response
+    <<-XML
+    <Message TypeCode="Response" MajorVersion="3" MinorVersion="4" FixVersion="0" xmlns="http://www.aafes.com/credit">
```

Comment:
This response should all be indented an additional tab to get ahead of [my RuboCop PR](https://github.com/activemerchant/active_merchant/pull/3685).